src/Snackbar/snackbarStyles.ts

```diff
@@ -0,0 +1,60 @@
+import styled from '@emotion/styled';
+
+export interface SnackbarPropsStyles {
+    anchorOrigin: { vertical: string; horizontal: string };
+    open: boolean;
+}
+
+export const TRANSITION_CLOSE_TIME = 0;
+
+export const SnackbarWrapper = styled.div(() => ({
+    '.content-snackbar-wrapper': {
+        display: 'flex',
+        alignItems: 'center',
+        marginTop: 4
+    },
+
+    '.snackbarOpen': {
+        minHeight: 40,
+        opacity: 1,
+        transform: `scale(${1.02}, ${1.02 ** 2})`,
+        transition: 'opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms, transform 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms',
+        margin: '20px'
+    },
+
+    '.snackbarClosed': {
+        minHeight: 40,
+        opacity: 0,
+        transition: 'opacity 0ms cubic-bezier(0.4, 0, 0.2, 1) 0ms, transform 130ms cubic-bezier(0.4, 0, 0.2, 1) 0ms',
+        transform: 'none'
+    }
+}));
+
+export const StyledSnackbarButton = styled.span(() => ({
+    display: 'inline-flex',
+    backgroundColor: '#316a8f',
```

Comment:
Please replace with a similar from the lakefrontColors or add this as a new lakefrontColor and utilize.